spec/models/waitlist_team_spec.rb

```diff
@@ -0,0 +1,38 @@
+describe WaitlistTeam do
+  let(:assignment) { build(:assignment, id: 1, name: 'assignment1') }
+  let(:participant) { build(:participant, user_id: 1) }
+  let(:user) { build(:student, id: 1, name: 'new_name', fullname: 'new name', participants: [participant]) }
+  let(:team) { build(:assignment_team, id: 1, name: 'team1', users: [user]) }
+  let(:team_user) { build(:team_user, id: 1, user: user) }
+
+  before(:each) do
+    allow(TeamsUser).to receive(:where).with(team_id: 1).and_return([team_user])
+  end
+
+  let(:topic1) { build(:topic, id: 1) }
+  let(:topic2) { build(:topic, id: 2) }
+  let(:waitlist_team1) { build(:waitlist_team) }
+  let(:waitlist_team2) { build(:waitlist_team) }
+  let(:waitlist_team3) { build(:waitlist_team) }
+  let(:signedupteam1) { build(:signed_up_team) }
+
+  describe 'CRUD WaitlistTeam' do
```

Comment:
Can you be more specific on what the individual test-suite do instead of CRUD waitlistteam ?